src/wallet/wallet.h

```diff
@@ -49,6 +50,8 @@ static const CAmount DEFAULT_TRANSACTION_MAXFEE = 0.1 * COIN;
 static const CAmount MIN_CHANGE = CENT;
 //! -txconfirmtarget default
 static const unsigned int DEFAULT_TX_CONFIRM_TARGET = 2;
+//! -optintofullrbf default
+static const bool DEFAULT_OPT_INTO_FULL_RBF = true;
```

Comment:
Could this make sense as a separate PR with `DEFAULT_OPT_INTO_FULL_RBF = false`?